lib/postoffice.ex

```diff
@@ -61,4 +61,19 @@ defmodule Postoffice do
   end
 
   def ping, do: Application.ensure_started(:postoffice)
+
+  def add_messages_to_deliver(messages) do
+    messages_number = Enum.count(messages["payload"])
```

Comment:
`Enum.count` looks very clear, but I'd prefer `Kernel.lengh`, it seems more performat when we handle just lists
[here](https://medium.com/@kana_sama/length-vs-amount-37a1c431141a) you can find a nice article about it

Reply:
After talk with you, finally we decided use `Enum.count` because `Kernel.length` doesn't work with structs, and this is necessary to check the payload